src/node/nodestate.h

```diff
@@ -282,11 +282,17 @@ namespace ccf
       }
       auto frontend = handler.value();
 
-      enclave::RPCContext ctx(
-        enclave::InvalidSessionId, node_cert, ccf::ActorsType::members);
+      const enclave::SessionContext node_session(
+        enclave::InvalidSessionId, node_cert);
+      auto ctx = enclave::make_rpc_context(node_session, packed);
+
       ctx.is_create_request = true;
 
-      frontend->process(ctx, packed);
+#ifdef PBFT
```

Comment:
Why this? The request should go via `process()` no matter the consensus.

Reply:
Yeah this was based on an old git branch, and it looks like a merge reintroduced this `ifdef` branch. Removed.